portfolio/src/main/webapp/script.js

```diff
@@ -23,28 +23,53 @@ function hover(){
 /**
   * Adds filler paragraph when ANY subparagraph is clicked.
    */
-var paragraphTitle
+var paragraphTitle;
+
+var aboutMe = "Hi! My name is Leah, and I’m a rising sophomore at Gettysburg College in "
+    + "their STEM Scholar program. I am a Computer Science major and Mathematics minor, and "
+    + "I am on track to complete the Engineering 4-2 program, and would like to pursue a "
+    + "career in Software Engineering. I am a member of Girls Who Code, and the National "
+    + "Center for Women & Information Technology. My favorite languages to work with are "
+    + "Java, Javascript and Python. "
+    + "\n\nCurrently, I am a Student Training in Engineering Program (STEP) intern at Google.";
+var pastProjects = `Schedify \n\tSocial Networking / Planner webapp using Google Calendar API, final project for Google Computer Science Summer Institute`
++"\n\tPYTHON HTML CSS "+
++"\n\tKevin Cam Michael Kelly"+
+"\nGStress"+
+"\n\tHealth and wellness website designed for students of Gettysburg College"+
+"\n\tHTML CSS JS"+
+"\n\tKarla Gonzalez"+
+"\nUnspoken"+
+"\n\tWebsite designed to teach Sign Language"+
+"\n\tMelissa Wilson Rebecca Kalapala Sabrina Ahmed Reina";
+var funFacts = " - I got into CS by doing robotics in middle school!"+
+    "\n - My favorite color is Yellow!"+
+    "\n - I can wiggle my ears and eyebrows!"+
+    "\n - I am the mother to several plants!"+
+    "\n - I’m very competitive at Pictionary!";
+
+
 function expand(info){
```

Comment:
can you use a more descriptive function name and add a comment or docstring above the function describing what it does?